README.md

```diff
@@ -75,18 +75,26 @@ COPY --chown=1001:0 <path_to_customized_snippet> /config/configDropins/overrides
 
 It is important to be able to observe the logs emitted by Open Liberty when it is running in docker. A best practice method would be to emit the logs in JSON and to then consume it with a logging stack of your choice.
 
-Configure your Open Liberty docker image to emit JSON formatted logs to the console/standard-out with your selection of liberty logging events by providing the following environment variables to your Open Liberty DockerFile.
-
-For example:
+Configure your Open Liberty docker image to emit JSON formatted logs to the console/standard-out with your selection of liberty logging events by creating and adding a `bootstrap.properties` file to your Open Liberty Dockerfile.
+```dockerfile
+COPY --chown=1001:0  bootstrap.properties /config/
 ```
-//This example illustrates the use of all available logging sources.
-ENV WLP_LOGGING_CONSOLE_FORMAT=JSON
-ENV WLP_LOGGING_CONSOLE_LOGLEVEL=info
-ENV WLP_LOGGING_CONSOLE_SOURCE=message,trace,accessLog,ffdc,audit
+Then include the following environment variables into the properties file you have just created. As an optional feature, you can decide whether or not to disable `messages.log` or `trace.log` output.
```

Comment:
bootstrap.properties contents aren't considered to be environment variables.  Perhaps say "Then include the following properties into the file you have just created.  You can also disable writing to the messages.log or trace.log files if you don't need them."